docker/development.yml

```diff
@@ -33,6 +33,7 @@ services:
       - ../config/nginx.conf.development.template:/etc/nginx/nginx.conf
     ports:
       - '8000:8000'
+      - '4631:4631'
```

Comment:
Can we use one port for everything? I don't think we need different ports for different apps.